libr/anal/type.c

```diff
@@ -146,20 +147,38 @@ R_API RList *r_anal_types_from_fcn(RAnal *anal, RAnalFunction *fcn) {
 	return uniq;
 }
 
-static void enum_type_fini(void *e, void *user) {
+/**
+ * @brief Function to free a RAnalBaseTypeEnum
```

Comment:
same

Reply:
from what i read in this comment i would expect r_anal_base_type_enum_free () as function name. fini is not free. also taking void* in a custom free helper is calling for problems

Reply:
Yeah but the free helper is for generic RVector, with typedef void (*RVectorFree)(void *e, void *user);